example/conc/README.md

```diff
@@ -28,7 +28,7 @@ $ owi conc ./mini.wat
 Trap: unreachable
 Model:
   (model
-    (symbol_1 (i32 6)))
+    (symbol_1 (i32.const 6)))
```

Comment:
Is this change expected?

Reply:
Yes, I'm using the `Concrete_value.pp` to avoid calling Z3 to fetch a model since the trace's `symbol_values` will be assignment we want to output.